go/libraries/doltcore/env/actions/staged.go

```diff
@@ -216,28 +221,19 @@ func getStagedAndWorking(ctx context.Context, dEnv *env.DoltEnv) (*doltdb.RootVa
 	return roots[StagedRoot], roots[WorkingRoot], nil
 }
 
-func getWorkingAndHead(ctx context.Context, dEnv *env.DoltEnv) (*doltdb.RootValue, *doltdb.RootValue, error) {
-	roots, err := getRoots(ctx, dEnv, WorkingRoot, HeadRoot)
-
-	if err != nil {
-		return nil, nil, err
-	}
-
-	return roots[WorkingRoot], roots[HeadRoot], nil
-}
-
-func getRoots(ctx context.Context, dEnv *env.DoltEnv, rootTypes ...RootType) (map[RootType]*doltdb.RootValue, error) {
+// Get all wanted roots.
```

Comment:
In golang always begin method doc with the name of the func, and use simple present tense verbs. So, "getRoots gets..."